### ISSUE TITLE
Nest API going away

### ISSUE THREAD

**Issue by freelancemke** [michael5r/mmm-nest-cameras#6]
Since the Works with Nest is going away and I can't get a Nest Token to make this work, do you have any plans on getting it to work with the google API they are now using?

**Comment by oasephiroth** [comment]
I'm also interested in if this will be updated. I took a quick look at how google was handling things now and it didn't look like there was still a way to accomplish setting up this module.

**Comment by DavidJacques01** [comment]
Same.  Tried to muddle through it but was unable to get a "pin" for the Oauth client.  Looking forward to setting up this module in the near future!

**Comment by JoeMansour2003** [comment]
I am also intressed in getting a new module that works with the Google api. Let me know if I can help in anyway.

**Comment by juniordevdenver** [comment]
Looks like he has another module mmm-nest-status that can authenticate with the google api.  I am really interested in making a magic mirror, so I might take a crack at making a PR to update.

**Comment by michael5r** [comment]
@charles-ryan-dawson I would appreciate that - just haven't had the time to do so. The Google API authentication is pretty straight-forward - it's the getting video feeds that's the issue.

**Comment by juniordevdenver** [comment]
Good to know.  I only have gotten as far as the auth.

**Comment by juniordevdenver** [comment]
@michael5r I was able to generate the RtspStream via postman. Looks like you get a 5 minute stream lifetime by default. Were your troubles with getting that RtspStream to show? Or was it more with dealing with keeping the stream alive?

**Comment by michael5r** [comment]
@charles-ryan-dawson Yeah, it wasn't working straight off the bat and then I didn't have time to go down the debugging rabbit hole.

**Comment by juniordevdenver** [comment]
Yeah this is a pretty deep rabbit hole from what I have found.  Found an interesting library that might be able to help.  

https://gitlab.jak.guru/jakg/nest-rtsp

**Comment by jamiesonpepper** [comment]
@michael5r I've been playing around with the Smart Device Management API and using Gemini as a crutch and managed to cobble together a working demo that pulls the first (and only) doorbell I have and streams it.

Your need to sign up for a Device Access Console project and get an Access Token with oAuth credentials configured in Google Cloud and that same oAuth client ID linked to the Device Access console but otherwise it seems to work.

For an easy way to get an Access Token you can use https://device-access-sample.web.app/ just make sure to add that URI to your   oAuth client configuration first.

I put the config.js module with placeholders at the link below:

https://codepen.io/Jamie-Pepper/pen/mydzjXe
